### DISCUSSION TITLE
How to exclude a Longhorn volume from a default recurring job?

### DISCUSSION THREAD

**Discussion by comminutus** [longhorn/longhorn#7689] [Q&A]
Hi there, I have a daily backup job that is assigned to the `default` group.  How can I exclude a specific volume from this backup job?  The data is ephemeral and not something I want to backup.

**Comment by c3y1huang** [comment]
How about assigning the volume with a non-existent recurring-job group, then removing the `default` group from the volume?
```shell
> k -n longhorn-system label volume <VOLUME_NAME> recurring-job-group.longhorn.io/no-recurring-job=enabled
> k -n longhorn-system label volume <VOLUME_NAME> recurring-job-group.longhorn.io/default-
```

**Reply by iitsdani** [reply]
Can you do this declaratively by any chance?
